parser.cpp

```diff
@@ -21,16 +21,13 @@ std::map<std::string, std::string> Parser::ParseJsonString(std::string StringToP
             j++;
         }
     }
-    std::string Str = StringToParse;
-    std::string name = "name";
-    std::string hp = "hp";
-    std::string dmg = "dmg";
+    std::string array[] = {StringToParse, "name", "hp", "dmg"};
     
-    Map["name"] = Parser::FindData(Str,name);
+    Map["name"] = Parser::FindData(array[0],array[1]);
 
-    Map["hp"] = Parser::FindData(Str,hp);
+    Map["hp"] = Parser::FindData(array[0],array[2]);
 
-    Map["dmg"] = Parser::FindData(Str,dmg);
+    Map["dmg"] = Parser::FindData(array[0],array[3]);
```

Comment:
Jaj! Én úgy gondoltam ám, hogy miután betetted a string-eket egy tömbbe, azon már végig lehet iterálni, és minden elemmel szépen egymás után lehet hívni a FindData függvényt, illetve kulcsként szolgálhat a Map-nél.

Reply:
Én pedig nem is tudom, hogy gondoltam így... Javítva!